### PR TITLE
French translation, first draft.

### PR DESCRIPTION
**Pull request by JulienPalard**

### CODE REVIEW

File: fr/index.org [hunk 1]
```diff
@@ -1,99 +1,173 @@
 #+title:     Org mode for Emacs -- Votre vie en texte brut
-#+author:    Carsten Dominik
-#+email:     carsten at orgmode dot org
+#+author:    Julien Palard
+#+email:     julien at python dot org
```
Comment (tecosaur) [inline fr/index.org]: Just a little thing, we're actually removing author/email info from all pages (cleaner metadata, and on a lot of pages attribution isn't clear cut).

Comment (JulienPalard) [reply fr/index.org]: Works for me.

**Comment by tecosaur** [comment]
@JulienPalard thanks for your help! It's much appreciated. Would you mind if I pinged you again after #3 is merged? It has a few minor changes to `index.org` which I forgot about when you were contacted (sorry).

**Comment by tecosaur** [comment]
@JulienPalard I've just merged #3. If you'd be willing to incorperated the changes into your translation that would be marvelous!
If not, I'll just take the wonderful work you've done so far :smiley: Thanks regardless.

**Comment by JulienPalard** [comment]
Synced with #3, hope it won't happen often, else we'll have to use something better than git diff to sync between translations, like `.po` files.

**Comment by tecosaur** [comment]
Bastien (org maintainer) and I have chatted about what we want the site to be. The idea is a small collection of static files that don't need updating. I'm hoping this will be the last change to the index file.

This looks great! I'll merge this in now. Thanks again for your work :)

**Pull request merged**